### DISCUSSION TITLE
Windows Defender

### DISCUSSION THREAD

**Discussion by ElliottS200** [security-mountain/security-mountain.github.io#6] [Ideas]
Following a previous discussion, if we delve into OS specifics, i'm thinking we include internal features that can be harnessed, such as Windows defender, rather than deploying external or generic tools? Thoughts?

**Comment by apollodelphi702** [comment]
Personally, I would argue that we should _prefer_ internal features (particularly where they provide similar or better functionality), as relying on third party tools less would theoretically increase the longevity of the advise and we needn't worry about vetting third party tools.
